cluster/examples/kubernetes/ceph/object-multisite-pull-realm.yaml

```diff
@@ -0,0 +1,53 @@
+#################################################################################################################
+# Create an object store with settings for a test environment. Only a single OSD is required in this example.
+#  kubectl create -f object-multisite.yaml
+#################################################################################################################
+apiVersion: ceph.rook.io/v1
```

Comment:
@travisn what do you think about putting a sample kubernetes secret yaml (similar to what's in the documentation) here as well?